src/controllers/__tests__/authenticationController.test.ts

```diff
@@ -206,13 +195,32 @@ const getClientProfileRedirectionUrl = (token: string): UrlFromString => {
   };
 };
 
-const controller = new AuthenticationController(
-  redisSessionStorage,
-  samlCert,
-  spidStrategyInstance,
-  tokenService,
-  getClientProfileRedirectionUrl
-);
+let controller: AuthenticationController;
+beforeAll(async () => {
+  if (!process.env.IDP_METADATA_URL) {
+    // TODO: Env variable will not load on local test env
```

Comment:
what do you mean?

Reply:
if I run tests locally with the command `yarn test` the env variable `IDP_METADATA_URL` is not visible even if it is defined inside `.env` file. Circle-CI test environment uses env variables defined inside `.circleci/config.yml`, local tests don't.